tests/utils/site-builder.js

```diff
@@ -150,6 +150,8 @@ const createSiteBuilder = ({ siteName }) => {
       return builder
     },
     buildAsync: async () => {
+      // Replace by Promise.allSettled() [Supported for >=12.9.0]
```

Comment:
`allSettled` is not the same as the this code.
We want to run the tasks in order, hence `await` inside the for loop

Reply:
Oh. Okay. Removing the comment.